lib/rules/visual-label-better-than-aria-suggestion-v9.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+var elementType = require("jsx-ast-utils").elementType;
+const { hasNonEmptyProp } = require("../util/hasNonEmptyProp");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+// Define an array of applicable component names
+const allowedComponents = ["Dropdown", "SpinButton", "CompoundButton"];
```

Comment:
Can we move this out into the applicableComponents folder? Also, Inputs and other types of buttons apply to this rule.